public/js/components/Scopes.js

```diff
@@ -72,13 +78,21 @@ function getScopes(pauseInfo, selectedFrame) {
     return null;
   }
 
-  let scope = selectedFrame.scope;
-  if (!scope) {
+  let selectedScope = selectedFrame.scope;
+
+  if (!selectedScope) {
     return null;
   }
 
   const scopes = [];
 
+  let scope = selectedScope;
+
+  function isScopeOfPauseFrame(innerScope) {
+    const pauseActor = pauseInfo.getIn(["frame", "scope"]).get("actor");
+    return innerScope.actor === pauseActor;
+  }
```

Comment:
Nit: I'd prefer if we got the `pausedScopeActor`at the top of `getScopes` and used it below:

```js
if (scope.actor === pausedScopeActor) {
}
```